clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -320,7 +327,7 @@ type podStartupLatency struct {
 
 // SummaryName returns name of the summary.
 func (p *podStartupLatency) SummaryName() string {
-	return podStartupLatencyMeasurementName
+	return p.summaryName
```

Comment:
Maybe sth like "podStartupLatencyMeasurementName + p.summaryName"

Reply:
If we change this we should remember to update perfdash (iirc it should be able to handle multiple parser for one metric). There was an assumption to not change summaries names, though.

Reply:
If we want to have two measurements of the same type (as here), they can't have the same name, right?

Reply:
You are right. I missed the metric adding part when I look at the PR.